docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -0,0 +1,431 @@
+        memoryAvailable: 1m30s
+        imageFSAvailable: 1m30s
+        imageFSInodesFree: 1m30s
+        nodeFSAvailable: 1m30s
+        nodeFSInodesFree: 1m30s
+      evictionHard:                            # hard, i.e. immediate eviction (used if the node is out of resources, avoiding the OS generally run out of resources fail processes indiscriminately)
+        memoryAvailable: 100Mi
+        imageFSAvailable: 5%
+        imageFSInodesFree: 5%
+        nodeFSAvailable: 5%
+        nodeFSInodesFree: 5%
+      evictionMinimumReclaim:                  # additional resources to reclaim after hitting the hard eviction thresholds to not hit the same thresholds soon after again
+        memoryAvailable: 0Mi
+        imageFSAvailable: 0Mi
+        imageFSInodesFree: 0Mi
+        nodeFSAvailable: 0Mi
+        nodeFSInodesFree: 0Mi
+      evictionMaxPodGracePeriod: 90            # caps pod's `terminationGracePeriodSeconds` value during soft evictions (general grace periods)
+      evictionPressureTransitionPeriod: 4m0s   # stabilization time window to avoid flapping of node eviction state
+```
+
+You can tweak these settings also individually per worker pool (`spec.provider.workers.kubernetes.kubelet...`), which makes sense especially with different machine types (and also workload that you may want to schedule there).
+
+Physical memory is not compressible, but you can overcome this issue to some degree (alpha since Kubernetes `v1.22` in combination with the feature gate `NodeSwap` on the `kubelet`) with swap memory. You can read more in this [introductory blog](https://kubernetes.io/blog/2021/08/09/run-nodes-with-swap-alpha) and the [docs](https://kubernetes.io/docs/concepts/architecture/nodes/#swap-memory). If you chose to use it (still only alpha at the time of this writing) you may want to consider also the risks associated with swap memory:
+
+- Reduced performance predictability
+- Reduced performance up to page trashing
+- Reduced security as secrets, normally held only in memory, could be swapped out to disk
+
+That said, the various options mentioned above are only remotely related to HA and will not be further explored throughout this document, but just to remind you: if a zone goes down, load patterns will shift, existing pods will probably receive more load and will require more resources (especially because it is often practically impossible to set "proper" resource requests, which drive node allocation - limits are always ignored by the scheduler) or more pods will/must be placed on the existing and/or new nodes and then these settings, which are generally critical (especially if you switch on [bin-packing for Gardener-managed clusters](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_scheduling_profiles.md) as a cost saving measure), will become even more critical during a zone outage.
```

Comment:
```suggestion
That said, the various options mentioned above are only remotely related to HA and will not be further explored throughout this document, but just to remind you: if a zone goes down, load patterns will shift, existing pods will probably receive more load and will require more resources (especially because it is often practically impossible to set "proper" resource requests, which drive node allocation - limits are always ignored by the scheduler) or more pods will/must be placed on the existing and/or new nodes and then these settings, which are generally critical (especially if you switch on [bin-packing for Gardener-managed clusters](shoot_scheduling_profiles.md) as a cost saving measure), will become even more critical during a zone outage.
```